### ISSUE TITLE
Fail to start notebook in windows subsystem for linux

### ISSUE THREAD

**Issue by mdai26** [computationalmodelling/fidimag#139]
Hi! 

I am trying to use fidimag in the windows subsystem for linux (WSL). But the notebook cannot be start normally. I don't know how to do next to connect to the notebook and do simulations.  

The output of my command is attached.

Best regards.

Minyi
![error_message](https://user-images.githubusercontent.com/45828456/59698051-83082280-91b4-11e9-9164-e57f6ddde0f0.PNG)

**Comment by rpep** [comment]
Hi Minyi,

It sounds like the port forwarding is not working on Windows. I am not sure how we can advise you to get this working; none of the developers are Windows users.

My suggestion would be to try and install Fidimag without Docker inside WSL - from my understanding this should work without any issues by just installing as on Ubuntu. The commands for doing this are:
```
git clone https://github.com/computationalmodelling/fidimag/
cd fidimag
sudo apt-get install -y -qq  build-essential cmake cython3 python3-dev python3-pip liblapack-dev libopenblas-dev
pip3 install numpy matplotlib cython ipywidgets pyvtk six psutil scipy pytest
./bin/install-sundials.sh
./bin/install-fftw.sh
make
```
On each run, or add to your ~/.bashrc file
```
export PYTHONPATH=path/to/fidimag:${PYTHONPATH}
export LD_LIBRARY_PATH=path/to/fidimag/local/lib:${LD_LIBRARY_PATH}
```
Then try running the tests:

```
py.test -v .
```

**Comment by mdai26** [comment]
Hi!

Thanks for your reply. I typed all the command as you said but I encountered problem like this:
![error_message2](https://user-images.githubusercontent.com/45828456/59720869-89fb5900-91e5-11e9-9789-d7d6750cf767.PNG)

I also received warnings about fortran compile when I type ./bin/install-sundials.sh and ./bin/install-fftw.sh. Do I need to install a fortran compile and then install Fidimag?

Best regards.

Minyi
